dev/integration_tests/ios_host_app/analysis_options.yaml

```diff
@@ -0,0 +1,5 @@
+# These dart files are copied into a newly created project and have
+# errors outside that context.
+analyzer:
+  exclude:
+    - 'flutterapp/lib/**'
```

Comment:
Hm, this isn't working, I'm still getting errors from these files.  You guys have any hints to get it to ignore `ios_host_app/flutterapp/lib/main.dart` and `ios_host_app/flutterapp/lib/marquee.dart`?

Reply:
how about `**/flutterapp/lib/**`?

Reply:
I think you were looking at an outdated diff.
Also `./dev/integration_tests/ios_add2app_life_cycle/flutterapp/lib` exists and is a real package.

Reply:
> Ugh, didn't even get to the analyzer this time. Still trying to figure out how to have floating dart test files that aren't really in a package, but will be copied into a newly created package by an integration test.
> 
> ```
> Warning: pubspec in /tmp/flutter sdk/dev/integration_tests/ios_host_app/flutterapp has out of date dependencies. Please run "flutter update-packages --force-upgrade" to update them correctly.
> ```

I don't think that test is even using the analyzer, it's just checking the pubspec checksum

Reply:
https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/commands/update_packages.dart#L180